### PR TITLE
Feat/implant results

### PR DESCRIPTION
**Pull request by stmharry**

### CODE REVIEW

File: scripts/commands.py [hunk 1]
```diff
@@ -272,39 +274,135 @@ def postprocess(
...
+                df_full_tooth["top"] = pd.Series.isin(
+                    df_full_tooth["fdi_int"] // 10, [1, 2]
+                )
+                df_full_tooth["left"] = pd.Series.isin(
+                    df_full_tooth["fdi_int"] // 10, [2, 3]
+                )
+                df_full_tooth["x"] = (df_full_tooth["fdi_int"] % 10 - 0.5) * np.where(
+                    df_full_tooth["left"], 1, -1
+                )
+                df_full_tooth["y"] = np.where(df_full_tooth["top"], 1, -1)
```
Comment (wahahahaya) [inline scripts/commands.py:314]: i don't get what's x and y means

File: scripts/commands.py [hunk 2]
```diff
@@ -272,39 +274,135 @@ def postprocess(
...
+                )
+
+                interp = scipy.interpolate.RBFInterpolator(
+                    y=df_full_tooth.loc[df_full_tooth["exists"], ["x", "y"]],
+                    d=df_full_tooth.loc[
+                        df_full_tooth["exists"], ["bbox_x_center", "bbox_y_center"]
+                    ],
+                    smoothing=1e0,
+                    kernel="thin_plate_spline",
+                )
```
Comment (wahahahaya) [inline scripts/commands.py:335]: 1. how edges are calculated.
2. If only a few teeth are present, the result can be unreliable.

File: scripts/commands.py (hunk 1 above)
Comment (stmharry) [reply scripts/commands.py:314]: I lay out teeth on a grid with 
- x values in (-7.5, -6.5, ..., -0.5, +0.5, ..., +7.5)
- y values in (-1, +1)

and then interpolate the bbox centers with this grid on missing teeth.

File: scripts/commands.py (hunk 2 above)
Comment (stmharry) [reply scripts/commands.py:335]: 1. Do you mean bbox? Bounding boxes are a result from the inference results.
2. True, so these cases will have to be investigated further.

Note that this function also does extrapolation aside from interpolation.

**Pull request merged**